catalog/generators/platform-thorntail/info.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "generator",
+  "name": "platform thorntail",
```

Comment:
Could we make this more like the Vert.x one? "Thorntail Base Platform"

Reply:
I copied from the node.js one, sure